### PR TITLE
Add article about alpine

### PR DESCRIPTION
**Pull request by hatchan**
This just adds the article, still needs to be added to ToC somewhere

### CODE REVIEW

File: content/docs/faq/alpine.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+tags: docker, alpine
+---
+
+## I receive a "context canceled" error when using an Alpine container
+
+This error occurs during the `setup environment` step, and is caused by the
+fact that we use `/bin/bash` to run the steps. Alpine doesn't contain the bash
+shell, so it fails. 
+
+Wercker doesn't required `bash`, is is also possible to use `sh`. You're able
```
Comment (Tonnu) [inline content/docs/faq/alpine.md]: doesn't require

File: content/docs/faq/alpine.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+```yaml
+box:
+    id: alpine
+    cmd: /bin/sh
+```
+## I receive the "Unable to sync environment" when using docker-push on Alpine
+
+After receiving this warning the container stops. Before running the docker
+deploy step, we first synchronyze the environment variables from within the
```
Comment (Tonnu) [inline content/docs/faq/alpine.md]: synchronize

**Comment by Tonnu** [comment]
Minor spelling, :+1:

**Comment by Tonnu** [comment]
ToC is here: https://github.com/wercker/docs/blob/master/docs-sidebar/toc-docs.json

**Comment by hatchan** [comment]
Updated the article with a TOC and added the faq to the index and TOC

**Comment by flenter** [comment]
lgtm, can this be merged in?

**Pull request merged**